src/components/sidebar.tsx

```diff
@@ -46,6 +46,7 @@ export const Sidebar = ({ data }: ChildProps): JSX.Element => {
         tree={tree}
         exclude={["/product/relay/"]}
       />
+      <DynamicNav root="integrations" title="Integrations" tree={tree} />
```

Comment:
so you can do root=product/integrations, and then in the product one you can use exclude (see platformSidebar.tsx for an example)

```
exclude={[`/product/integrations/`]}
```

Reply:
Nice! I'll make that change today.